src/vendor/github.com/xelabs/go-mysqlstack/sqlparser/sql.y

```diff
@@ -1265,9 +1275,15 @@ show_statement_type:
     }
   }
 | non_reserved_keyword
-{
+  {
```

Comment:
Format

Reply:
> Format

The format here has no problem, please check again.
It was not aligned before, I just fix it.

Reply:
Just mean to be aligned.